Source/Core/Core/Boot/Boot.cpp

```diff
@@ -226,7 +226,11 @@ bool CBoot::BootUp()
 	NOTICE_LOG(BOOT, "Booting %s", _StartupPara.m_strFilename.c_str());
 
 	g_symbolDB.Clear();
-	VideoInterface::Preset(_StartupPara.bNTSC);
+
+	// PAL Wii uses NTSC framerate and linecount in 60Hz/480i and 60Hz/480p modes
+	const bool bPAL60 = _StartupPara.bWii && (SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.E60") || SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.PGS"));
```

Comment:
Ok, in this case, i vote for allowing 576p in this pr, and let somebody make a new pr that changes the gui, and makes 576p impossible. For example via radio button: pal50<->pal60<->progressive.